tests/drivers/dma/chan_blen_transfer/src/main.c

```diff
@@ -14,35 +14,27 @@
 #include <zephyr.h>
 #include <ztest.h>
 
-extern void test_dma_m2m_chan0_burst8(void);
```

Comment:
Commit message is again not true. It is doing more than adapting the test for stm32 dma. Split the patch in 2, one that introduce the generic changes, one for the stm32 dma

Reply:
changed into different commits 17c3e68416dda8fa668c4f90110516f090651257 and eacaa587d30892a0aaa4e15ae04b9d8b867139e8